common/services/profile/profile.unformat.js

```diff
@@ -0,0 +1,59 @@
+const getAnswer = (profile, field) => {
```

Comment:
I think it would be nice to try and stick to either a folder or a file rather than maintaining both. I find having both creates extra confusion as there are now two places to look.

We did it for person with the vision to change it which I understand with other pressures we never got round to. So I'd rather not do the same if we can avoid it...

Reply:
I was doing it like person precisely for that reason!